src/javascript/jsdoc.ts

```diff
@@ -156,7 +156,8 @@ export function parseJsdoc(docs: string): Annotation {
       docs, {unwrap: false, lineNumbers: true, preserveWhitespace: true});
   // Strip any leading and trailing newline characters in the
   // description of multiline comments for readibility.
-  const description = d.description && d.description.replace(/^\n+|\n+$/g, '');
+  const description =
+      d.description && d.description.replace(/^\n+|\n+$/g, '').trim();
```

Comment:
trim was supposedly left out on purpose to preserve indentation on the first line